src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
It was also still defined in `lib/tags/index.js`

I've raised #217 to address that.